packages/@adobe/spectrum-css-temp/vars/spectrum-dark.css

```diff
@@ -297,7 +297,7 @@
   --spectrum-well-background-color: rgba(235,235,235,0.02);
   --spectrum-well-border-color: rgba(255,255,255,0.05);
   /* react spectrum additions */
-  --react-spectrum-datepicker-placeholder-color: var(--spectrum-global-color-gray-600); /* 4.8:1 contrast ratio */
+  --react-spectrum-datepicker-placeholder-color: var(--spectrum-global-color-gray-600); /* 6.04:1 contrast ratio */
```

Comment:
We could probably get rid of this variable entirely now and just inline `var(--spectrum-gray-600)` into the date picker CSS since they are all the same across all themes?